### PR TITLE
Marharyta k

### PR DESCRIPTION
**Pull request by marharyta-k**

### CODE REVIEW

File: MarharytaKalynchuk/HW3/task3.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+a = 9000000
```
Comment (kolyasalubov) [inline MarharytaKalynchuk/HW3/task3.py:1]: one way of implementation:

a, b = b, a

Comment (marharyta-k) [reply MarharytaKalynchuk/HW3/task3.py:1]: Thank you for your comments! 
Yes, I remember now this solution from the lecture.

Comment (kolyasalubov) [reply MarharytaKalynchuk/HW3/task3.py:1]: No at all)!

File: MarharytaKalynchuk/HW4/task1.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+number = int(input('Please enter your number: '))
```
Comment (kolyasalubov) [inline MarharytaKalynchuk/HW4/task1.py:1]: good job

File: MarharytaKalynchuk/HW5/binary_decimal_code_runner.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+def binary(decimal):
```
Comment (kolyasalubov) [inline MarharytaKalynchuk/HW5/binary_decimal_code_runner.py:1]: good job

File: MarharytaKalynchuk/HW5/booleans.py [hunk 4]
```diff
@@ -0,0 +1,8 @@
+def integer_boolean(binary_number):
```
Comment (kolyasalubov) [inline MarharytaKalynchuk/HW5/booleans.py:1]: good job

File: MarharytaKalynchuk/HW5/count_vowels.py [hunk 5]
```diff
@@ -0,0 +1,6 @@
+def count_vowels(word):
```
Comment (kolyasalubov) [inline MarharytaKalynchuk/HW5/count_vowels.py:1]: good job

File: MarharytaKalynchuk/HW5/isogram.py [hunk 6]
```diff
@@ -0,0 +1,13 @@
+def is_isogram(word):
```
Comment (kolyasalubov) [inline MarharytaKalynchuk/HW5/isogram.py:1]: good job

File: MarharytaKalynchuk/HW5/task1.py [hunk 7]
```diff
@@ -0,0 +1,17 @@
+ints = [2, 7, 9, 5, 7]
```
Comment (kolyasalubov) [inline MarharytaKalynchuk/HW5/task1.py:1]: good job

File: MarharytaKalynchuk/HW5/task2.py [hunk 8]
```diff
@@ -0,0 +1,11 @@
+
+def fibo():
```
Comment (kolyasalubov) [inline MarharytaKalynchuk/HW5/task2.py:2]: good job

File: MarharytaKalynchuk/HW6/coderunner5.py [hunk 9]
```diff
@@ -0,0 +1,7 @@
+def nth_smallest(lst, n):
```
Comment (kolyasalubov) [inline MarharytaKalynchuk/HW6/coderunner5.py:1]: good job

File: MarharytaKalynchuk/HW6/task1.py [hunk 10]
```diff
@@ -0,0 +1,24 @@
+even_numbers = []
+odd_numbers = []
+numbers_not_div_2_3 = []
+
+
```
Comment (kolyasalubov) [inline MarharytaKalynchuk/HW6/task1.py:5]: good job

**Pull request closed without merging**